Makefile

```diff
@@ -8,8 +8,8 @@ all: protoc install test
 NOVENDOR = go list github.com/tendermint/abci/... | grep -v /vendor/
 
 install-protoc:
-	# Download: https://github.com/google/protobuf/releases
-	go get github.com/golang/protobuf/protoc-gen-go
+# Download: https://github.com/google/protobuf/releases
```

Comment:
?

Reply:
This was in the original PR. I’ve reverted that line (will push in a moment).